spec/requests/comments_spec.rb

```diff
@@ -1,7 +1,7 @@
-require 'rails_helper'
+# require 'rails_helper'
 
-RSpec.describe 'Comments', type: :request do
-  describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
-  end
-end
+# RSpec.describe 'Comments', type: :request do
+#   describe 'GET /index' do
+#     pending "add some examples (or delete) #{__FILE__}"
+#   end
+# end
```

Comment:
- [ ] It's good option to remove commented code from your file. Kindly delete them if you do not want to.

Reply:
Alright, thank you for this tip. I have removed the commented-out code.